srcpkgs/goimapnotify/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'goimapnotify'
+pkgname=goimapnotify
+version=2.0
+revision=1
+_gitlab_rev="6bbf63dd50a024251b6fe271287a4db389e73e43"
```

Comment:
if you use `https://gitlab.com/shackra/goimapnotify/-/archive/${version}/${pkganme}-${version}.tar.gz` as the distfile, the git revision won't be included in the extracted path and this custom variable won't be necessary.